docs/webhooks.rst

```diff
@@ -58,7 +58,9 @@ GitHub
   *application/x-www-form-urlencoded* work
 * Leave the **Secrets** field blank
 * Select **Let me select individual events**,
-  and mark **Branch or tag creation**, **Branch or tag deletion** and **Pushes** events
+  and mark **Branch or tag creation**, **Branch or tag deletion** and **Pushes** events.
+  If you want to use Read the Docs to :doc:`preview documentation changes from pull
+  requests </pull-requests>`, also mark **Pull requests** events.
```

Comment:
```suggestion
  and mark **Branch or tag creation**, **Branch or tag deletion**, **Pushes** and **Pull requests** events.
```

I think it's better to just mark those without too many explanations. Otherwise, people will need to modify the webhook if they want that feature later.